### PR TITLE
Bugs/fix some failing system tests

### PR DESCRIPTION
**Pull request by abelikt**
## Proposed changes

Fix some of the currently failing tests.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: tests/PySys/tedge/call_tedge_-h/run.py [hunk 1]
```diff
@@ -27,5 +27,5 @@ def execute(self):
 
     def validate(self):
         self.assertGrep("tedge.out", "USAGE:", contains=True)
-        self.assertGrep("tedge.out", "FLAGS:", contains=True)
+        self.assertGrep("tedge.out", "OPTIONS:", contains=True)
```
Comment (PradeepKiruvale) [inline tests/PySys/tedge/call_tedge_-h/run.py:30]: Did not get why FLAGS replaced by OPTIONS.

File: tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py [hunk 2]
```diff
@@ -33,6 +33,3 @@ def validate(self):
             "tedge_connect_c8y.out", "tedge-agent service successfully started and enabled!", contains=True
         )
 
-        self.assertGrep(
```
Comment (PradeepKiruvale) [inline tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py:36]: Don't we need to check for the tedge-mapper?

File: tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_alarms/run.py [hunk 3]
```diff
@@ -25,6 +25,10 @@ def execute(self):
         # Solution: only inject the variables we really need
         environ={ 'HOME':os.environ.get('HOME')}
 
+        # TODO to wait for 5s until everythig is settled in the background is probably to
+        # long -> test and reduce
+        time.sleep(5)
```
Comment (matthiasbeyer) [inline tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_alarms/run.py:30]: May I ask why this is necessary in the first place? What needs to settle in the background exactly?

**Comment by abelikt** [comment]
This should have been a draft PR. Strangely, I cannot switch back to draft.

File: tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py (hunk 2 above)
Comment (abelikt) [reply tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py:36]: We should, I fixed the filename instead.

**Comment by abelikt** [comment]
Disabled test tedge_mapper_c8y_alarms due to https://github.com/thin-edge/thin-edge.io/issues/966

**Review by rina23q — APPROVED** [review]
Looks good, however, you need to rebase commits before you merge the PR.

**Comment by abelikt** [comment]
Rebased on main

**Comment by abelikt** [comment]
... after all that time this became a two-liner.
Rebased it on main.

**Pull request merged**